doc/content/language.md

```diff
@@ -1325,3 +1325,33 @@ output.icecast(%mp3, host="localhost", port=8000,
 ```
 
 so that passwords are not shown in the main script.
+
+### Code comments
+
+Comments can be added to your code in two ways:
+
+_Multi-line comments_ are comments that can span multiple lines. They are delimitated
+by the sequence of characters `#<` at the beginning and `>#` at the end. Anything
+in between those two sequences is considered code comment. Nested code comments
+are also accepted:
+
+```liquidsoap
+#<------- BEGIN CODE COMMENT ----#
+    This is a comment
+
+# This is also a comment
+
+  #< This is a nested code comment >#
+
+#--------- END CODE COMMENT ----->#
+```
```

Comment:
Maybe the simple syntax for multiline comments should be shown first?
````
Simple multiline comments
```liquidsoap
#<
  This is a comment
>#
```
Multiline comments could be nested
```liquidsoap
#<
  This is a top-level comment
  # This is also a comment
  #<
    This is a nested code comment
  ># 
>#
```
Fancy looking multiline comment
```liquidsoap
#<------- BEGIN CODE COMMENT ----#
Comments can also look like this
#--------- END CODE COMMENT ----->#
```
````